components/skin/ui/src/main/resources/PhenoTips/ContentTopMenu.xml

```diff
@@ -667,26 +667,32 @@
     </property>
     <property>
       <code>document.observe("xwiki:dom:loaded", function () {
-  var exportLinks = $$('.editbody .export-link');
+  var exportLinks = $$(' .export-link');
   exportLinks.invoke('observe', 'click', function (event) {
     event.stop();
-    new XWiki.widgets.ConfirmationBox(
-      {
-        onYes : function() {
-          var exportAfterSave = function() {
-            window.open(event.findElement().href,"_blank");
-            document.stopObserving('xwiki:document:saved', exportAfterSave);
-          };
-          document.observe('xwiki:document:saved', exportAfterSave);
-          document.fire('xwiki:actions:save', {'continue' : true, 'form' : event.findElement('form')});
-        },
-        onNo : function() {
-          window.open(event.findElement().href,"_blank");
-        }
-      },
-      /* Interaction parameters */
-      { confirmationText: "$services.localization.render('phenotips.contentTopMenu.saveBeforeExport')", showCancelButton: true }
-    );
+	if (typeof event.findElement().up('.editbody') !== "undefined") {
```

Comment:
You can just say [`event.findElement('.editbody')`](http://api.prototypejs.org/dom/Event/prototype/findElement/)